pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci</groupId>
         <artifactId>jenkins</artifactId>
-        <version>1.54</version>
+        <version>1.76</version>
```

Comment:
the pom file wouldn't build on Java 11 without some updates.

I didn't bother going to latest which requires Java 11 as we still want to use this library on 8 for a bit for ATH.
I know I could add some trickery but I don't think needed for this repo.

Reply:
Yeah it is deprecated anyway.